models/post/index.js

```diff
@@ -53,9 +53,20 @@ async function findOne( id ) {
 }
 
 
-async function findAll() {
+async function findAllUser( user ) {
   try {
-    const posts = await DB.query('SELECT post.*, user.first_name, user.last_name FROM `post` INNER JOIN `user` ON post.user_id = user.id ORDER BY `date` DESC');
+    const posts = await DB.query('SELECT post.*, user.first_name, user.last_name FROM `post` INNER JOIN `user` ON post.user_id = user.id WHERE ? ORDER BY `date` DESC', user);
```

Comment:
Update to be string template:
```
`SELECT post.*, user.first_name, user.last_name FROM \`post\` INNER JOIN \`user\` ON post.user_id = ${id} WHERE ? ORDER BY \`date\` DESC
```
That way query only has 1 argument and you're able to read it inline.